### PR TITLE
add discovery flag to network.toml

### PR DESCRIPTION
**Pull request by jerry-yu**
add discovery flag to network.toml

### CODE REVIEW

File: scripts/create_cita_config.py [hunk 1]
```diff
@@ -417,6 +419,12 @@ def parse_arguments():
         action='store_true',
         help='The data is encrypted and transmitted on the network')
 
+    # enable encrypted
```
Comment (kaikai1024) [inline scripts/create_cita_config.py]: enable discovery

File: scripts/create_cita_config.py [hunk 2]
```diff
@@ -162,6 +162,7 @@ def __init__(self, chain_name, output_dir):
                                              'authorities.list')
         self.nodes = NetworkAddressList()
         self.enable_tls = False
+        self.enable_discovery = False
```
Comment (leeyr338) [inline scripts/create_cita_config.py]: I think it better set `enable discovery` as default.

Comment (leeyr338) [reply scripts/create_cita_config.py]: And this will cause `discovery test` failed.

**Comment by jerry-yu** [comment]
there maybe some proplem when enable disvovery

**Pull request closed without merging**